### ISSUE TITLE
Go quiz with impossible question

### ISSUE THREAD

**Issue by jordan-bonecutter** [Ebazhanov/linkedin-skill-assessments-quizzes#6951]
I'm submitting an ISSUE: please check one with "x"

- [ ] New question{'s} 
- [ ] New assessment 
- [x] Missed questions/answers 
- [ ] Request for new quiz/answers/...
- [ ] Documentation issue or request for ...
- [ ] ...

The following question:

#### Q6. Which is the _only_ valid import statement in Go?

- [ ] &shy;

```go
      import "github/gin-gonic/gin"
```

- [ ] &shy;

```go
      import "https://github.com/gin-gonic/gin"
```

- [ ] &shy;

```go
      import "../template"
```

- [x] &shy;

```go
      import "github.com/gin-gonic/gin"
```

[Import in GoLang](https://golangdocs.com/import-in-golang)

has 2 correct answers. Technically, the first one is also correct for a package named github (go mod init github). Inside this package there could be a subdirectory gin-gonic/gin. This could then be imported with the given import statement.

**Comment by haxkd** [comment]
No, the import statement import "github/gin-gonic/gin" is not correct in Go. It's missing the .com part in the domain.

**Comment by jordan-bonecutter** [comment]
No, I can do:


`go mod init github `

Then I can make the subdirectory:



`gin-gonic/gin`

Now the code in that directory will be importable with github/gin-gonic/gin

**Comment by chrishalbert** [comment]
While that's a clever way to make the specific case work, that won't _always_ be valid, and if its not always valid, it's invalid - think about a tautology from discrete mathematics.

**Comment by jordan-bonecutter** [comment]
> While that's a clever way to make the specific case work, that won't _always_ be valid, and if its not always valid, it's invalid - think about a tautology from discrete mathematics.

You can say the same about the correct answer. If the owner takes the repository down it will no longer be a valid import. Both are circumstantial.

**Comment by chrishalbert** [comment]
Today it is valid though. If the day comes that the repository is renamed/archived, yes, you will then be correct at that point in time. If you argued this question on an exam, I think the professor would give you at least partial credit, but I think its pretty cut and dry here. Appreciate the outside the box thinking though!

**Comment by jordan-bonecutter** [comment]
> Today it is valid though. If the day comes that the repository is renamed/archived, yes, you will then be correct at that point in time. If you argued this question on an exam, I think the professor would give you at least partial credit, but I think its pretty cut and dry here. Appreciate the outside the box thinking though!

IMO a bad question. It requires knowing that this repository exists to properly answer. And again, today my answer is correct. No difference
